tabbycat/draw/generator/powerpair.py

```diff
@@ -285,15 +285,15 @@ def get_n_teams(self, teams: list['Team']) -> int:
         # necessary for enumerated seed values
         return max([t.subrank for t in teams if t.subrank is not None])
 
-    def assignment_cost(self, t1, t2, size, bracket=None):
+    def assignment_cost(self, t1, t2, size, bracket=None) -> Optional[int]:
         penalty = super().assignment_cost(t1, t2, size)
         if penalty is None:
             return None
 
         # Add penalty for seeing the pullup again
         has_pullup = 'pullup' in self.team_flags.get(t1, []) or 'pullup' in self.team_flags.get(t2, [])
         if self.options["pullup_debates_penalty"] and has_pullup:
-            penalty += max(t1.pullup_debates, t2.pullup_debates) * self.options["pullup_debates_penalty"]
+            penalty += (t1.pullup_debates + t2.pullup_debates) * self.options["pullup_debates_penalty"]
```

Comment:
This isn't exactly what you're looking for; this isn't a pullup penalty _per se_, but so that a team doesn't get put against the pullup team too often (an APDA rule; don't think there's an equivalent in WSDC)